### PR TITLE
Remove lisk-ui as submodule - Closes #458

### LINKED ISSUE #458: Decouple and remove Lisk UI
To coincide with the planned replacement of `lisk-ui` with the scheduled `1.0.0` release of `lisk-nano`, we should remove and decommission the use of `lisk-ui` as a submodule, including any API related logic.

### PR DESCRIPTION
**Pull request by diego-G**
This PR allows us to get rid of Git submodules. The main implication is Lisk core removes its GUI entirely. From now on, Lisk Nano becomes the unique GUI available and supported.

Closes #458

### CODE REVIEW

**Comment by karmacoma** [comment]
@diego-G You will need to also review the `grunt release` task here: https://github.com/LiskHQ/lisk/blob/development/Gruntfile.js#L36

**Comment by karmacoma** [comment]
@diego-G PR title is misleading as your commits also close #622. Please address separately.

File: Gruntfile.js [hunk 1]
```diff
@@ -40,7 +40,6 @@ module.exports = function (grunt) {
 						util.format('mkdir -p %s', version_dir),
 						util.format('mkdir -p %s/logs', version_dir),
 						util.format('mkdir -p %s/pids', version_dir),
-						util.format('mkdir -p %s/public', version_dir),
```
Comment (karmacoma) [inline Gruntfile.js]: As discussed, we need this: `util.format('mkdir -p %s/public/dapps', version_dir),`

File: Gruntfile.js [hunk 2]
```diff
@@ -49,13 +49,6 @@ module.exports = function (grunt) {
...
 						util.format('cp %s/sql/migrations/*.sql %s/sql/migrations/', __dirname, version_dir),
-						util.format('cd %s/public && mkdir -p ./static', __dirname),
-						'npm install && bower install && grunt release && cd ../',
-						util.format('cp %s/public/wallet.html %s/public/', __dirname, version_dir),
-						util.format('cp %s/public/loading.html %s/public/', __dirname, version_dir),
-						util.format('cp -Rf %s/public/images %s/public/', __dirname, version_dir),
-						util.format('cp -Rf %s/public/partials %s/public/', __dirname, version_dir),
-						util.format('cp -RfL %s/public/static %s/public/', __dirname, version_dir),
 						util.format('mkdir -p %s/public/node_modules', version_dir),
 						util.format('cp -Rf %s/public/node_modules/chart.js %s/public/node_modules', __dirname, version_dir),
```
Comment (SargeKhan) [inline Gruntfile.js]: Do we need the rest of the `/public/node_modules/` related lines here?

**Pull request merged**